app/models/label/LabelTable.scala

```diff
@@ -81,6 +83,15 @@ object LabelTable {
   val neighborhoods = regions.filter(_.deleted === false).filter(_.regionTypeId === 2)
 
 
+  val anonId = "97760883-8ef0-4309-9a5e-0c086ef27573"
```

Comment:
Ah I see, as in querying the user table for the id associated with the username "anonymous" to get that id?